functests/utils/nodes/nodes.go

```diff
@@ -195,6 +195,11 @@ func BannedCPUs(node corev1.Node) (banned cpuset.CPUSet, err error) {
 		return cpuset.NewCPUSet(), fmt.Errorf("failed to execute %v: %v", cmd, err)
 	}
 
+	if bannedCPUs == "" {
+		testlog.Infof("Banned CPUs on node %q returned empty set", node.Name)
+		return cpuset.NewCPUSet(), nil // TODO: should this be a error?
```

Comment:
I think we should return an error, otherwise, it will fail during the test check.

Reply:
The more I think about it, the more I agree. Fixing.

Reply:
I thought more, and I think that we should not fail here. We should fail one level up. From the perspective of this function having empty banned cpu string is legal and means "no cpus are banned".

Reply:
I just afraid that it will harder to debug the failing test. But in general, both approaches work for me.